cpp/dolfinx/fem/assemble_vector_impl.h

```diff
@@ -347,13 +346,14 @@ void _lift_bc_interior_facets(
 
   // Temporaries for joint dofmaps
   std::vector<std::int32_t> dmapjoint0, dmapjoint1;
+  assert(facets.size() % 4 == 0);
+  std::array<std::int32_t, 2> cells;
```

Comment:
The two `std::array`s should be inside the loop.

Reply:
I've now moved them. Out of curiosity, why do we want to reallocate them inside the loop?

Reply:
They're statically allocated, so there is no overhead. Code is simpler and shorter with allocation inside.